README.md

```diff
@@ -1,17 +1,8 @@
 # Deis Monitor v2
-
-[![Build Status](https://travis-ci.org/deis/monitor.svg?branch=master)](https://travis-ci.org/deis/monitor)
-[![Docker Repository on Quay](https://quay.io/repository/deisci/monitor/status "Docker Repository on Quay")](https://quay.io/repository/deisci/monitor)
```

Comment:
Well they will need to for config options and such. Didnt want to pollute the main readme with all that